### DISCUSSION TITLE
Disable auto-fit in the viewer

### DISCUSSION THREAD

**Discussion by BBBBBbruce** [libigl/libigl#1929] [Q&A] [answered]
Hello, 

I wonder how can I view the mesh as the size I defined. 

What I have is a .msh file, I load it, and use the example in 110, I pass the V and F to set_mesh in the viewer. But when I multiply the scaler matrix for example MatrixXd(10,10,10)(before set_mesh()), I check the V, it has been enlarged, but the result shown in the viewer window is still the same as before enlargement.

My guess is the viewer has a fit-the-screen function, I wonder if I could disable it without changing the source code?

Thanks for the help.

**Comment by BBBBBbruce** [comment]
I found in viewer.core() has a function align_camera_center(), that is called also in launch_init(). I will just comment out it in the launch_init() at the moment

**Comment by alecjacobson** [comment, marked as answer]
yeah, that's the default behavior. I think you could load your mesh after
initialization to avoid that.
The camera controls are a bit hairy... Another hack would be to load
another mesh of a bounding box that you'd like to center the camera on.
